src/mappings/core.ts

```diff
@@ -8,8 +14,25 @@ import {
   Remove,
 } from "../types/PrimitiveEngine/PrimitiveEngine";
 import { toDecimal } from "../utils/decimals";
+import { updatePoolDayData, updatePoolHourData } from "./dayUpdates";
 
 export function handleAllocate(event: Allocate): void {
+  let allocate = new AllocateEntity(
+    event.transaction.hash.toHexString() +
+      "#" +
+      event.params.poolId.toHexString()
+  );
+  if (allocate) {
+    allocate.underlyingTokenAmount = event.params.delRisky;
+    allocate.quoteTokenAmount = event.params.delStable;
+    allocate.liquidityAmount = event.params.delLiquidity;
+    allocate.pool = event.params.poolId.toHexString();
+    allocate.timestamp = event.block.timestamp.toI32();
+    allocate.sender = event.transaction.from.toHexString();
```

Comment:
This wont correctly track sender because `sender` will always be the manager.  Therefore it's better to track allocate events on the manager.